### PR TITLE
(#141) Adding navbar and footer to login pages

### LINKED ISSUE #141: connect login/register page with header and footer
Header must have appropriate navbar links

### PR DESCRIPTION
**Pull request by shub-garg**

### CODE REVIEW

File: src/services/tests.py [hunk 1]
```diff
@@ -1484,7 +1484,7 @@ def test_get_pending_approval_services_client_error(self, mock_boto_resource):
         mock_table.scan.side_effect = ClientError(error_response, "Scan")
 
         services = self.service_repo.get_pending_approval_services()
-        self.assertNotEqual(services, [])
+        self.assertEqual(services, [])
```
Comment (ahhcash) [inline src/services/tests.py:1487]: why this change?

**Review by ahhcash — CHANGES_REQUESTED** [review]

File: src/services/tests.py (hunk 1 above)
Comment (ahhcash) [reply src/services/tests.py:1487]: maybe get rid of this test, looks flaky

**Review by ahhcash — APPROVED** [review]
as discussed, will be addressed in a future change

**Review by omerfarukbasar — APPROVED** [review]
Looks nice! Thumbs up emoji
